frontend/src/components/layout/NavBar.jsx

```diff
@@ -18,7 +18,7 @@ import Link from './Link.jsx'
 const links = [
   {
     name: `settings`,
-    link: `/oauth/myapps`,
```

Comment:
@hueyy @anushks, noting my comment above about the incorrect regex in our Python code, is this doing what we expect from a React PoV?

Reply:
These values go into the href attributes of anchor tags. No regex at work here, so looks fine.